docs/user_manual/calculations.md

```diff
@@ -108,6 +110,16 @@ Based on the requirements of observability mentioned above, user needs to satisf
 - Any sensor on a `Branch` for all branches: Parallel branches with either side of measurements count as one.
 - All `Branch3` sensors.
 
+##### Sufficient observability condition
+
+The condition check above only checks the necessary condition for observability. When the measurements are not independent enough, the system may still be unobservable even if the necessary condition is met.
+It is rather complicated to do a full sufficient and necessary observability check in generic cases. However, `power-grid-model` performs the sufficient condition check when the following conditions are met:
+
+1. The system is a radial network.
+2. The system does not have voltage phasor measurements.
+
+In this case, the validation of the independent measurements is rather straightforward. If the system is not observable, the calculation will raise a not observable error instead of sparse matrix error.
```

Comment:
```suggestion
In this case, the validation of the independent measurements is rather straightforward. If the system is not observable, the calculation will raise a `NotObservableError` instead of `SparseMatrixError`.
```